app/javascript/stylesheets/workTypes.scss

```diff
@@ -3,13 +3,22 @@
     text-decoration: none;
 
     &:after {
-      content: '\f078';
-      font-family: Fontawesome;
+      background-repeat: no-repeat;
+      background-size: 1rem;
+      content: "";
+      display: inline-block;
+      width: 1rem;
+      height: 1rem;
+      margin-left: 0.5rem;
+      background-image: escape-svg($accordion-button-active-icon);
       padding-left: 0.5rem;
+      padding-bottom: 0;
+      margin-left: 0.7rem;
```

Comment:
how to square this and what's on line 12?

Reply:
Oh, that's just a mistake. Don't need.

Reply:
Pick one and remove the other?